lab1.js

```diff
@@ -134,10 +156,24 @@ var tooHungryDay;
 */
 
 // your code goes here
-
+var appetite = function() {
+  var foodToday = 0;
+  var aveArray = [];
```

Comment:
this works...and it's possible to do this without needing a second array. It involves an if and a break.